### ISSUE TITLE
Style with leaflet kml

### ISSUE THREAD

**Issue by zcemycl** [aviklai/react-leaflet-kml#18]
I am trying to change the fillColor of the kml component.
```
{Kml && <ReactLeafletKml kml={Kml} 
                        style={{fillColor:'green',
                        fillOpacity: 0.1}}
                        />}
```
But it does not work like geojson.
```
<GeoJSON key="my-geojson" data={geojsonLondon}
                        style={{fillColor:'blue',
                                fillOpacity: 0.1}}/>
```
Are there any solves?

**Comment by aviklai** [comment]
@zcemycl Hi, can you please create a codesandbox with this issue?

**Comment by zcemycl** [comment]
@aviklai Thanks for the reply. I did look into those examples before, and tried to add tags in it. 
For example, https://github.com/andreafalzetti/uk-counties-list/blob/master/uk-areas/kml/England/England_East_of_England.kml
```
const doc = parser.parseFromString(kmlText,'text/html');
const polystyle = doc.createElement("PolyStyle");
const color = doc.createElement("color");
color.innerHTML = "colorcode";
polystyle.appendChild(color);
const y = document.getElementsByTagName("Placemark")[0]
```
Within placemark's style, I added polystyle color before passing into kml. But it does not work.

**Comment by aviklai** [comment]
@zcemycl Can you try:
<PolyStyle>
        <color>7dff0000</color>
      </PolyStyle>

**Comment by zcemycl** [comment]
@aviklai No. It does not work.

**Comment by aviklai** [comment]
@zcemycl Is this what you are trying to achieve?
![image](https://user-images.githubusercontent.com/8260668/119222082-40b39280-bafb-11eb-902a-0a6275cb5767.png)
If so - check this updated kml (I changed to file extension to txt to be able to upload it here):

[england.txt](https://github.com/aviklai/react-leaflet-kml/files/6526173/england.txt)

**Comment by zcemycl** [comment]
Yes and no. I do want to colorize the kml file, but do not want online conversion of kml files to another format like json or txt. 

I wonder if there is any easy method to make many conversions, like having 100 kml files. 

Anyway, thanks for your help.

**Comment by aviklai** [comment]
@zcemycl just to clarify - I didn't convert the file - I just changed the extension so I could upload it here, as github doesn't allow uploading a file with the lmk extension.
